db/revision.go

```diff
@@ -97,7 +97,7 @@ func (body Body) ShallowCopy() Body {
 
 func (body Body) DeepCopy() Body {
 	var copiedBody Body
-	base.DeepCopyInefficient(&copiedBody, body)
+	_ = base.DeepCopyInefficient(&copiedBody, body)
```

Comment:
Log Info on failure seems appropriate, as I think it would be unexpected.

Reply:
Fixed in latest revision